api/all/src/main/java/io/opentelemetry/api/OpenTelemetry.java

```diff
@@ -39,6 +40,11 @@ static OpenTelemetry propagating(ContextPropagators propagators) {
   /** Returns the {@link TracerProvider} for this {@link OpenTelemetry}. */
   TracerProvider getTracerProvider();
 
+  /** Returns the {@link MeterProvider} for this {@link OpenTelemetry}. */
```

Comment:
I think this is a good use of default methods for API evolution. But I'm wondering separate from that whether I should add default to the other methods for consistency and because a single-signal OpenTelemetry seems to be a generally useful concept. Not for this PR I think but something we could do.

Reply:
That would be:
```
ContextPropagators getPropagators();
TracerProvider getTracerProvider();
```
I don't have any aversion to adding default methods for all, but a user could effectively do this already by just providing noop implementations for the signals they don't need. Would improve the ergonomics and consistency though.